network/src/proto/network.proto

```diff
@@ -34,8 +34,13 @@ message DiscoveryMsg { repeated Note notes = 1; }
 
 // Identity message exchanged as part of the Identity protocol.
 message IdentityMsg {
+  enum Role {
```

Comment:
@phoenix-antigravity : Should we move role completely to proto enum? This diff creates redundancy b/w rust enum RoleType and proto enum Role defined here